migrations/20200624121732_consent.ts

```diff
@@ -32,7 +32,7 @@ export async function up (knex: Knex): Promise<void | Knex.SchemaBuilder> {
         return knex.schema.createTable('Consent',
           (t: Knex.CreateTableBuilder): void => {
             // TODO: Confirm string length for status and revoked at
-            t.string('id', 32).primary().notNullable()
+            t.string('id', 36).primary().notNullable()
```

Comment:
We technically should never modify a migration, but considering this isn't live anywhere yet it should be ok.

Reply:
Ok, but as per API definition consentId type is CorrelationId that is  always 36 characters long

Reply:
I agree with @sridharvoruganti  -> we should add and keep `id` constraint

Reply:
but I also agree with @lewisdaly that we shouldn't touch migration code -> but how to properly handle if migration code contains such a bug like here @lewisdaly ?

Reply:
You make a new migration :)